src/storm/solver/MinMaxLinearEquationSolver.cpp

```diff
@@ -147,7 +147,8 @@ std::vector<uint_fast64_t> const& MinMaxLinearEquationSolver<ValueType>::getInit
 
 template<typename ValueType>
 MinMaxLinearEquationSolverRequirements MinMaxLinearEquationSolver<ValueType>::getRequirements(
-    Environment const&, boost::optional<storm::solver::OptimizationDirection> const& direction, bool const& hasInitialScheduler) const {
+    Environment const&, [[maybe_unused]] boost::optional<storm::solver::OptimizationDirection> const& direction,
```

Comment:
This is now inconsistent; the environment is unnamed, yet the direction is "maybe unused"

Reply:
I did notice this unnamed parameter but was confused as to why it was unnamed while the other parameters were not so before I'd change something that was done on purpose, I just listed this in my "should be decided on by someone who knows what is going on here" markdown document